db/dcrpg/internal/txstmts.go

```diff
@@ -61,6 +61,8 @@ const (
 	SelectTxByHash       = `SELECT id, block_hash, block_index, tree FROM transactions WHERE tx_hash = $1;`
 	SelectTxsByBlockHash = `SELECT id, tx_hash, block_index, tree FROM transactions WHERE block_hash = $1;`
 
+	SelectTxBlockTimeByHash = `SELECT block_time FROM transactions where tx_hash = $1`
```

Comment:
This query should also be wrong too @chappjc. How do I best select the block time here taking into consideration that tx_hash may not be unique?

Reply:
Take the latest block time.  :)

Reply:
ok.

Reply:
LIMIT 1 ORDER BY block_time DESC

or something like that

Reply:
The same solution should perfectly work here 
```SelectAddressByTxHash = select id, address, value from addresses 
    where tx_hash=$1 and is_funding = true;```

Reply:
This should work seamlessy 
```SelectTxBlockTimeByHash = SELECT block_time FROM transactions where tx_hash = $1 ORDER BY block_time DESC LIMIT 1;
```

Reply:
But this is a query in the addresses table, not the transactions table.  You'd need the output index to get a single row, the one you want I should say

Reply:
Ah, yes.  I see you changed to the transactions table.  I gotta go, but we'll chat later